apprenda55-express/azuredeploy.json

```diff
@@ -0,0 +1,239 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01-preview/deploymentTemplate.json",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "platformAdminFirstName": {
+            "type": "string",
+            "metadata": {
+              "Description": "This is the first name of the platform administrator."
```

Comment:
Please change these "Description" to lowercase "description". Ignore the bot message for this change, we're fixing the bot